src/main/resources/swagger/api-docs.yaml

```diff
@@ -641,6 +643,102 @@ paths:
           schema:
             $ref: '#/definitions/ErrorReport'
 
+  '/proxy/{googleProject}/{clusterName}':
```

Comment:
I'm wondering how we should document the `/rstudio`, `/jupyter` APIs. Should we put them in swagger, or treat them as more "dynamic"? Should Leo have some kind of landing page that displays the set of tools running on a cluster?

Not sure atm - just posing questions.

Reply:
because the field will end up being even more dynamic than just rstudio and jupyter i'd rather we not have them documented in swagger. a landing page or at least some endpoint that users can call to get info about the running tools seems like a good approach IMO

Reply:
True, but since the tools are kind of baked in today it might be nice to document them in swagger? When we have fully dynamic tools then maybe we can look into different ways of discovering them.

Maybe we could have something like the following in swagger:
- `/proxy/{googleProject}/{clusterName}/jupyter` -- access Jupyter
- `/proxy/{googleProject}/{clusterName}/jupyter/lab` -- access JupyterLab
- `/proxy/{googleProject}/{clusterName}/rstudio` -- access RStudio
- `/proxy/{googleProject}/{clusterName}/setCookie` -- sets a cookie (I don't think this is tool specific)
- `/proxy/invalidateToken` -- invalidates token (again not tool specific)

WDYT?